src/pages/using-spark/guides/validation-messaging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+  title: Validation Messaging
+---
+import validationAnatomy from '../../../images/guides/validation-anatomy.png';
+import validationAnatomy2x from '../../../images/guides/validation-anatomy@2x.png';
+export const validationAnatomySet = `${validationAnatomy} 560w, ${validationAnatomy2x} 1120w`
+
+import {
+  SprkHeading,
+} from '@sparkdesignsystem/spark-react';
+
+<span id="validation-messaging-guide" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h1" additionalClasses="sprk-u-mbl sprk-u-Measure" isPageTitle>
+  Validation Messaging Guide
+</SprkHeading>
+
+<span id="main-takeaways" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Main Takeaways
+</SprkHeading>
+
+Giving validation feedback is an emotional process. Support users with this process by following these guidelines:
+
+- Explain what the Input expects before the user enters data. (Example: password rules, ranges, etc)
+- Give clear and helpful feedback when encountering a problem.
+- Use a combination of [placement](#placement), [timing](#timing), and [language](#language) to guide users through moments of uncertainty.
+
+<img
+  srcSet={validationAnatomySet}
+  class="sprk-u-Measure sprk-u-pam sprk-u-pbh"
+  src={validationAnatomy}
+  alt="Anatomy of Input validation with numbers pointing to each part: placement, timing, and language."
+/>
+
+
+<span id="placement" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Placement
+</SprkHeading>
+
+- Place messaging close to the Input. Spark places validation directly under the Input.
+- Provide a clear visual to indicate something has gone wrong with user Input and needs attention.
+- Plan error messaging for every field.
+- On forms with multiple fields, show validation messaging for each Input field to guide users to a successful submission.
+
+<span id="timing" class="docs-b-Link--anchor"></span>
+<SprkHeading element="h2" additionalClasses="sprk-u-mbm sprk-u-Measure" variant="displayThree">
+  Timing
+</SprkHeading>
+
+-	Show validation messaging after the user enters the data or when the Input becomes out of **selection focus**.
+- On forms, show error messaging **before** the user submits the form.
```

Comment:
```suggestion
- Show validation messaging after the user enters the data or when the Input becomes out of **selection focus**.
- On forms, show error messaging **before** the user submits the form.
```

Reply:
There was an extra space.

Reply:
weird! I wonder if prettier did this 
![image](https://user-images.githubusercontent.com/4342363/111695945-08958800-880a-11eb-86e3-81c0ae54bde7.png)